src/filesystem/implementations/local.h

```diff
@@ -60,12 +60,45 @@ class LocalFileSystem : public FileSystem {
   Status MakeTemporaryDirectory(
       std::string dir_path, std::string* temp_dir) override;
   Status DeletePath(const std::string& path) override;
+
+ private:
+  inline std::string getOSValidPath(const std::string& _path);
+  const char* kWindowsLongPathPrefix = "\\\\?\\";
```

Comment:
```suggestion
  const char* windows_long_path_prefix_ = "\\\\?\\";
```
To match Triton member variable convention.

Reply:
But that's a constant, not a variable? AFAIK, constants should be named like the former, not the latter.

Reply:
It *might* be more idiomatic to use `static constexpr char* kWindowsLongPathPrefix = "\\\\?\\";` to drive the fact this is, indeed, a constant.

Reply:
Had to verify with the google style [guide](https://google.github.io/styleguide/cppguide.html#Constant_Names) and you're both correct. I also found an instance in Triton code where we initialize to a constant [here](https://github.com/triton-inference-server/server/blob/main/src/command_line_parser.h#L85), however, we omit the leading "k", which is technically wrong according to the style standards.

We can defer to Nico's suggestion instead. I will update my comment above to that effect.